lp-to-jira

```diff
@@ -62,95 +201,70 @@ Examples:
         action='store_true'
     )
 
-    opts, args = opt_parser.parse_args()
-
-    # Make sure there's 2 arguments
-    if len(args) < 2:
-        opt_parser.print_usage()
-        return 1
+    opt_parser.add_option(
+        '-s', '--sync_project_bugs',
+        dest='sync_project_bugs',
+        action='store',
+        type=str,
+        help='Adds all bugs from a specified LP Project to specified Jira board'
+            ' if they are not already on the Jira board.'
+            ' Use --days to narrow down bugs'
+    )
 
-    bug_number = args[0]
-    project_id = args[1]
+    opt_parser.add_option(
+        '-d', '--days',
+        dest='days',
+        action='store',
+        type=int,
+        help='Only look for LP Bugs in the past n days'
+    )
 
-    jira_token_file = "{}/.jira.token".format(os.path.expanduser('~'))
+    opt_parser.add_option(
+        '--no-lp-tag',
+        dest='no_lp_tag',
+        action='store_true',
+        help='Do not add tag to LP Bug'
+    )
 
-    # 1. Connect to Launchpad API
+    opts, args = opt_parser.parse_args()
 
+    # Connect to Launchpad API
     # TODO: catch exception if the Launchpad API isn't open
     lp = Launchpad.login_with('foundations', 'production', version='devel')
 
-    # 2. Make sure the bug ID exist
-    bug = None
-    bug_pkg = None
-    try:
-        bug = lp.bugs[bug_number]
-    except Exception:
-        print("Couldn't find the Launchpad bug {}".format(bug_number))
-        return 1
-
-    if len(bug.bug_tasks) == 1:
-        bug_pkg = bug.bug_tasks[0].bug_target_name.split()[0]
-    else:
-        for task in bug.bug_tasks:
-            if "(Ubuntu)" in task.bug_target_name:
-                bug_pkg = task.bug_target_name.split()[0]
-
-    # 3. Connect to the JIRA API
+    # Connect to the JIRA API
     api = jira_api()
     jira = JIRA(api.server, basic_auth=(api.login, api.token))
 
-    # 4. Make sure that there's no bug in JIRA with the same ID
-    # than the Bug you're trying to import
-    existing_issue = jira.search_issues(
-        "project = \"{}\" AND summary ~ \"LP#{}\"".format(project_id, bug.id))
+    if opts.sync_project_bugs:
+        bugs_list = getAllLpProjectBugs(lp, opts.sync_project_bugs, opts.days)
 
-    if existing_issue:
-        print("Launchpad Issue {} is already logged "
-              "in JIRA here {}/browse/{}".format(
-                  bug_number,
-                  api.server,
-                  existing_issue[0].key))
-
-        if opts.exists:
-            # we are simply testing if the bug was already in JIRA
-            # if it was we return 0
-            return 0
+        for bug in bugs_list:
+            bug_number = bug.bug_link.split('/')[-1]
+            print(bug_number)
```

Comment:
Is this intended to not attempt to import the new bugs in JIRA at this point or was this for debugging purpose?

Reply:
Bleh! Yeah left over from debugging. :)